soccer/Referee.hpp

```diff
@@ -243,9 +160,31 @@ class Referee : public QThread {
     // team names in the packet.
     bool _isRefControlled = false;
 
-    float ballPlacementx;
-    float ballPlacementy;
+    float ballPlacementX;
+    float ballPlacementY;
 
 private:
     GameState _game_state;
+
+    static constexpr size_t RecvBufferSize =
+        std::numeric_limits<uint16_t>::max() + 1;
+    std::array<char, RecvBufferSize> _recv_buffer;
```

Comment:
Why this particular size?

Reply:
This was what it was before, I'm not sure what the rationale for what it was before is.